### PR TITLE
[2015.10.11]

### PR DESCRIPTION
**Pull request by ubless607**
Solved: https://sleazyfork.org/en/forum/discussion/6429/x

### CODE REVIEW

File: HandyImage.user.js [hunk 1]
```diff
@@ -817,6 +817,10 @@
 // @match		*://*.imgflash.net/img-*
 // @match		http://*.sexyimagexxx.com/img-*
 // @match   		http://*.imgtornado.com/img-*
+// @match   		http://*.img-365.com/image/*
+// @match   		http://*.imgbonk.com/image/*
+// @match   		http://*.imageyo.ga/image/*
+// @match   		http://*.daily-img.com/image/*
```
Comment (Owyn) [inline HandyImage.user.js]: wrong identation.

File: HandyImage.user.js [hunk 2]
```diff
@@ -817,6 +817,10 @@
 // @match		*://*.imgflash.net/img-*
 // @match		http://*.sexyimagexxx.com/img-*
 // @match   		http://*.imgtornado.com/img-*
+// @match   		http://*.img-365.com/image/*
+// @match   		http://*.imgbonk.com/image/*
+// @match   		http://*.imageyo.ga/image/*
```
Comment (Owyn) [inline HandyImage.user.js]: wrong indentation.

File: HandyImage.user.js [hunk 3]
```diff
@@ -817,6 +817,10 @@
 // @match		*://*.imgflash.net/img-*
 // @match		http://*.sexyimagexxx.com/img-*
 // @match   		http://*.imgtornado.com/img-*
+// @match   		http://*.img-365.com/image/*
+// @match   		http://*.imgbonk.com/image/*
```
Comment (Owyn) [inline HandyImage.user.js]: wrong indentation.

File: HandyImage.user.js [hunk 4]
```diff
@@ -817,6 +817,10 @@
 // @match		*://*.imgflash.net/img-*
 // @match		http://*.sexyimagexxx.com/img-*
 // @match   		http://*.imgtornado.com/img-*
+// @match   		http://*.img-365.com/image/*
```
Comment (Owyn) [inline HandyImage.user.js]: wrong indentation.

File: HandyImage.user.js (hunk 1 above)
Comment (ubless607) [inline HandyImage.user.js]: any tips how?,.,.

**Comment by Owyn** [comment]
@ubless607 either press "TAB" button twice or copy from previous lines

**Pull request merged**